charts/filebrowser/README.md

```diff
@@ -0,0 +1,11 @@
+## Filebrowser - Helm chart for Kubernetes Deployment
+
+This is a basic helm chart that allows to run Filebrowser full node on a Kubernetes cluster.
+
+Filebrowser can be deployed using the following steps:
+helm install filebrowser ./ \
+--set DOCUMENTSERVER_URL="URL to the documentserver filebrowser can use"
+--set THREEBOT_APP_ID="Hostname where the application will be exposed" 
+
+Important note:
+Make sure to expose the deployed solution on default webserver ports  (80 or 443)
```

Comment:
can u plz write detailed steps in readme? 
I didn't get what u mean with 
`Make sure to expose the deployed solution on default webserver ports  (80 or 443)`
u mean to create a service to expose that pod on 80 or what?
Also charts in this repo should work behind traefik as a load balancer not node port
check the [ingress part](https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/gitea/values.yaml#L30)  and [service part ](https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/gitea/values.yaml#L16)
so I recomment that u test it in similar setup like on a vdc or digital ocean machine with k3s installed https://k3s.io/

Reply:
On the jumpscale (with filtst) applications were exposed on an FQDN. The application does a redirect for authorization. When the app is exposed to another port the redirect won't work as the javascript isn't taking into account the port number. If the helm packages will be exposed on an FQDN (port 80/443) there is no problem.

Reply:
can u plz make use of ur ingress.yaml? ingress is responsible for exposing the solution enable it in ur values.yaml 
I need one command to test this chart like the following example
```
helm install monitoring marketplace/monitoring_stack  --set prometheus.ingress.hosts[0]=ashraf-monitoring.webg2dev.grid.tf 
```
after running this command you will get monitoring stack running and exposed on port 443
this section in readme is important to have ssl certificate on ur domain
![Screenshot from 2021-01-05 16-57-02](https://user-images.githubusercontent.com/1689456/103661224-32634080-4f77-11eb-9cf9-5f3b23c756cf.png)
for more details refere to 
https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/monitoringstack/values.yaml#L91
https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/monitoringstack/values.yaml#L1557
https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/monitoringstack/templates/prometheus/ingress.yaml#L14
https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/gitea/values.yaml#L16